lmfdb/static/lmfdb.js

```diff
@@ -722,3 +688,31 @@ function hide_schema(tbl) {
   $("#"+tbl+"-schema-shower").show();
   return false;
 }
+
+
+/* add handler for search forms to clean their own
+   form data and remove keys for empty (default) values */
+$(document).ready(function () {
+  let forms = [document.getElementById('search'),
+               document.getElementById('re-search')];
```

Comment:
There are several examples where you deleted `cleanSubmit` but the form was named something else (e.g. `hgmsearch` or `knowl-search`).  You should either rename the forms or add other options here.

Reply:
Thanks, i think renaming the search forms makes more sense, or we could add a class `clean` that the js uses to find which forms to clean? Downside of that is people have to remember to add it, if all search forms want this functionality anyways naming them all search seems simpler

Reply:
I think just renaming the forms to `search` or `re-search` in each case makes sense.  If people want to use this functionality on another kind of form, they can add the id to the list here.